decentraland/starter.js

```diff
@@ -29,22 +13,22 @@ const startExpo = () => {
 };
 
 const start = async () => {
+  const config = process.env.CONFIG;
```

Comment:
> Does the command-line arg make it through as an env var?
Yes.

> I think we should log around here whether it's booting up in `testnet` or `local` mode.
Ok, I'll do that.

Reply:
> > Does the command-line arg make it through as an env var?
> 
> Yes. Will be better if we use a cmd arg ?

How exactly would this look if we did it?

It might be clearer to a reader looking at this script first that we aren't heavily using vanilla env vars if we wrote it to explicitly use a command-line arg, but I'm not sure how to do that off the top of my head. I guess we could use that [`yargs`](https://www.npmjs.com/package/yargs) package I mentioned?

> 
> > I think we should log around here whether it's booting up in `testnet` or `local` mode.
> 
> Ok, I'll do that.

Great, thanks!

Reply:
Yes, `yargs` seems a good choice for that, here is an example of how we can use it for this case (based on the project github example):
```javascript
#!/usr/bin/env node
const argv = require('yargs').argv;
const config = argv.network;
```

```sh
./starter.js --network testnet
```

Reply:
I'm not sure if we need to run the javascript as a exec file I'll need to do some tryouts.

Reply:
Okay, sounds good

Reply:
Done.

Reply:
Thanks! Resolving